doc/getting_started/installation_win.rst

```diff
@@ -339,7 +340,7 @@ To build for the ARM-based Nordic nRF52 Development Kit:
 This should check that all the tools and toolchain are set up correctly for
 your own Zephyr development.
 
-Using Windows 10 WSL (Windows Subsystem for Linux)
+Option 3: Windows 10 WSL (Windows Subsystem for Linux)
```

Comment:
The title underline must be at least as long as the title, so add some more ``====``'s

Reply:
Done